### ISSUE TITLE
Add unit tests for a `Frame` element

### ISSUE THREAD

**Issue by mvanzulli** [mvanzulli/ONSAS.jl#391]
These functions should be tested with IO:

- [ ] internal_forces

**Comment by mforets** [comment]
Right, though indirectly it is what we are testing by the current method. Maybe it should be made more direct. 

Otherwise, Is there some ONSAS.m tests that you would recommend using?
